environments/prod.yml

```diff
@@ -13,3 +13,10 @@ prometheusRules:
     annotations:
       description: "High number of bad responses coming from selected-tests service"
       summary: "Selected tests service giving high number of bad responses over the last hour (current number: {{ $value }}). More information can be seen here: https://grafana.corp.mongodb.com/d/zenZa6oWk/selected-tests-service?orgId=4"
+resources: # Burstable
+  requests: # guaranteed amount of resources
+    cpu: "5m"
+    memory: "30Mi"
```

Comment:
I know very little about what are standard cpu/memory requests, but 5m cpu and 30Mi memory look low to me. (I say this only because the [kanopy docs](https://github.com/10gen/kanopy-docs/blob/master/docs/production_checklist.md#resource-requestslimits) put 100m cpu and 192Mi memory in their example.) Might we set these to 100m cpu and 192Mi memory, just to be safe?

Reply:
From [Grafana](https://grafana.corp.mongodb.com/d/zenZa6oWk/selected-tests-service?orgId=4), 100m cpu and 192Mi would be over provisioning by about 50x (at the moment). The request resources used by the web app are incredibly low right now.

I would like to provision the application low and increase it later if there is an issue.

I don't think there is any harm in setting these values lower than the suggested values (as they are also multiples of the actual required resources). It may also help us validate that the application limits are being enforced.

Reply:
But we don't have any alerts set up on application limits right now, so we won't know right away if there is an issue. I'd prefer to overprovision up front since it is a web app and traffic will increase once this project is live, but I do not feel super strongly about this, so am fine with it as-is.